domain/src/main/java/com/thoughtworks/go/config/materials/git/GitMaterial.java

```diff
@@ -199,95 +185,6 @@ public ValidationBean handleException(Exception e, GitVersion gitVersion) {
         }
     }
 
-    private GitCommand getGit(File workingdir, int preferredCloneDepth, SubprocessExecutionContext executionContext) {
```

Comment:
Most of the changes here are IntelliJ IDEA cleanup; grouping non-public methods at the bottom and inlining some unchanging parameters.

Keep scrolling for the main bits; I'll call out what the essence of this file's changes are.